### ISSUE TITLE
pscoast -M headers

### ISSUE THREAD

**Issue by joa-quim** [GenericMappingTools/gmt#2750]
Kind of warnings too. Currently when extracting countries it adds headers like
```
$ gmt coast -EIS -M | head
> Extract data for Iceland Segment 0
337.454017769   64.7773278112
337.443057282   64.7777778482
337.444406265   64.7827282561
337.455366752   64.7792779718
337.454017769   64.7773278112
> Extract data for Iceland Segment 1
337.506459482   64.7676270119
337.513710266   64.7745775846
337.516070986   64.7724274074
```
but that ``Extract data for`` is useless info. Better to just print ``Iceland-seg_0`` or something of this kind. I am thinking that when we implement choropleth maps a unique identifier will be needed for each country, and the simpler we make it now the better than.

**Comment by PaulWessel** [comment]
Agree on the useless part.  Not sure if appending -seg_? is wise since for some countries you will have tons of islands and you are unlikely to want separate colors.  Perhaps add **-Z**_ISOcode_ to the header in addition to the plain text.

**Comment by joa-quim** [comment]
OK, fine with that.

**Comment by joa-quim** [comment]
Fixed in #2751
